app/scripts/lib/account-tracker.js

```diff
@@ -336,26 +335,29 @@ export default class AccountTracker {
    */
   async _updateAccountsViaBalanceChecker(addresses, deployedContractAddress) {
     const { accounts } = this.store.getState();
-    this.web3.setProvider(this._provider);
-    const ethContract = this.web3.eth
-      .contract(SINGLE_CALL_BALANCES_ABI)
-      .at(deployedContractAddress);
-    const ethBalance = ['0x0'];
-
-    ethContract.balances(addresses, ethBalance, (error, result) => {
-      if (error) {
-        log.warn(
-          `MetaMask - Account Tracker single call balance fetch failed`,
-          error,
-        );
-        Promise.all(addresses.map(this._updateAccount.bind(this)));
-        return;
-      }
+    this.ethersProvider = new ethers.providers.Web3Provider(this._provider);
+
+    const ethContract = await new ethers.Contract(
+      deployedContractAddress,
+      SINGLE_CALL_BALANCES_ABI,
+      this.ethersProvider,
+    );
+    const ethBalance = ['0x0000000000000000000000000000000000000000'];
```

Comment:
why so many additional `0`s here?

Reply:
Ethers packge won’t work with 0x0 as zero adress. 
This is soultion that @danjm suggested to me.

Reply:
👍 ethers has this in as a constant in `@ethersproject/constants` but looks like we don't use it anywhere else so lets not just pull it in for this.